src/gui/optionsdialog.cpp

```diff
@@ -493,6 +496,37 @@ void OptionsDialog::initializeLanguageCombo()
     }
 }
 
+void OptionsDialog::initializeThemeCombo()
+{
+    m_ui->comboTheme->addItem(tr("Default"));
+    const QString customUIThemePath = Preferences::instance()->customUIThemePath();
+    if (!customUIThemePath.isEmpty())
+        m_ui->comboTheme->addItem(Utils::Fs::toNativePath(customUIThemePath));
+    m_ui->comboTheme->insertSeparator(m_ui->comboTheme->count());
+    m_ui->comboTheme->addItem(tr("Select..."));
+    m_ui->comboTheme->setCurrentIndex(Preferences::instance()->useCustomUITheme() ? 1 : 0);
+
+    connect(m_ui->comboTheme, qOverload<int>(&QComboBox::currentIndexChanged), this, [this](const int index)
+    {
+        if (index == (m_ui->comboTheme->count() - 1)) {
```

Comment:
I would prefer return early on opposite condition. You produced a needless block of indention currently.